From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Sun, 30 Oct 2011 22:18:39 +0100
Subject: [PATCH] Don't use -Werror

Remove the -Werror flag to compile it with gcc4. Yes I know, a real fix
different ;)

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
---
 Rules.make |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Rules.make b/Rules.make
index d8ed6a0..a002453 100644
--- a/Rules.make
+++ b/Rules.make
@@ -6,7 +6,7 @@ HOSTCC = gcc
 IFLAGS = -I$(TOPDIR)/include
 #DFLAGS = -g
 OFLAGS = -O3 -fomit-frame-pointer
-CFLAGS = -Wall -Werror $(IFLAGS) $(DFLAGS) $(OFLAGS)
+CFLAGS = -Wall $(IFLAGS) $(DFLAGS) $(OFLAGS)
 
 SRCS += $(wildcard *.c)
 OBJS += $(subst .c,.o,$(SRCS))
